web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v443/migrate-default.sql

```diff
@@ -1,2 +1,5 @@
 UPDATE Settings SET value='4.4.3' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
+
+
+ALTER TABLE spg_page ADD COLUMN access_expression VARCHAR(2048);
```

Comment:
The majority of the tables/columns are lowercase without any understore (_) separator. I suggest removing the name.
```suggestion
ALTER TABLE spg_page ADD COLUMN accessexpression VARCHAR(2048);
```

Reply:
I have refactored the name to be accessexpression